api/src/main/java/brooklyn/mementos/BrooklynMementoPersister.java

```diff
@@ -97,10 +97,13 @@
     /** @deprecated since 0.7.0, use {@link #checkpoint(BrooklynMementoRawData, PersistenceExceptionHandler)} 
      * and javadoc on implementations of that */ @Deprecated  // pretty sure this is not used outwith deprecated code
     void checkpoint(BrooklynMemento memento, PersistenceExceptionHandler exceptionHandler);
-    
-    void checkpoint(BrooklynMementoRawData newMemento, PersistenceExceptionHandler exceptionHandler);
 
+    /** applies a full checkpoint (write) of all state */  
+    void checkpoint(BrooklynMementoRawData newMemento, PersistenceExceptionHandler exceptionHandler);
+    /** applies a partial write of state delta */  
     void delta(Delta delta, PersistenceExceptionHandler exceptionHandler);
+    /** inserts an additional delta to be written on the next delta request */
+    void queueDelta(Delta delta);
```

Comment:
We already have the `PeriodicDeltaChangeListener`, which wraps calls to `persister.delta`. It handles queues, and does smart stuff like if you do a modify then a remove within the same time period, it will discard the modify and just do the remove.

Should we try to reuse `PeriodicDeltaChangeListener` somehow, rather than having `queueDelta` here?